src/actions/ActionHandlers.ts

```diff
@@ -0,0 +1,47 @@
+import { TasksState, NewTaskState } from 'src/reducers/types'
+import { Actions } from 'src/actions/types'
+import { Task } from 'src/types/global'
+
+const _ = require('lodash')
+
+export const getTasks = (state: TasksState, action: Actions) => {
+  return Object.assign({}, state, {
+    tasks: _.get(action, 'payload.tasks', ''),
+  })
+}
+
+export const addTask = (state: TasksState, action: Actions) => {
+  return Object.assign({}, state, {
```

Comment:
also another way to do this is
`return {
    ...state,
    tasks: [...state.tasks, _.get(action, 'payload.task', [])],
    error: {},
}`